portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -275,3 +277,16 @@ function drawStateIncarcerationChart() {
       document.getElementById('chart-container'));
   chart.draw(stateData, stateChartDimensions);
 }
+
+async function loadChartData() {
+  const response = await fetch('/data');
+  const dataPairs = await response.json();
+  const chartData = [];
+
+  dataPairs.forEach((pair) => {
+    const tempArray = [pair.group, pair.percent];
+    chartData.push(tempArray);
```

Comment:
Personally I'm not fond of tempArray as a variable name here. I'd just pass the list directly to push:

chartData.push([pair.group, pair.percent])

Reply:
Okay sure, I'll incorporate that.